pkg/standalone/standalone.go

```diff
@@ -1298,68 +1343,33 @@ func copyWithTimeout(ctx context.Context, dst io.Writer, src io.Reader) (int64,
 	}
 }
 
-// getPlacementImageName returns the resolved placement image name for online `dapr init`.
+// getDaprImageName returns the resolved Dapr image name for online `dapr init`.
 // It can either be resolved to the image-registry if given, otherwise GitHub container registry if
 // selected or fallback to Docker Hub.
-func getPlacementImageName(imageInfo daprImageInfo, info initInfo) (string, error) {
+func getDaprImageName(imageInfo daprImageInfo, info initInfo) (string, error) {
 	image, err := resolveImageURI(imageInfo)
 	if err != nil {
 		return "", err
 	}
 
-	image, err = getPlacementImageWithTag(image, info.runtimeVersion, info.imageVariant)
+	image, err = getDaprImageWithTag(image, info.runtimeVersion, info.imageVariant)
 	if err != nil {
 		return "", err
 	}
 
 	// if default registry is GHCR and the image is not available in or cannot be pulled from GHCR
 	// fallback to using dockerhub.
 	if useGHCR(imageInfo, info.fromDir) && !tryPullImage(image, info.containerRuntime) {
-		print.InfoStatusEvent(os.Stdout, "Placement image not found in Github container registry, pulling it from Docker Hub")
-		image, err = getPlacementImageWithTag(daprDockerImageName, info.runtimeVersion, info.imageVariant)
+		print.InfoStatusEvent(os.Stdout, "Image not found in Github container registry, pulling it from Docker Hub")
+		image, err = getDaprImageWithTag(daprDockerImageName, info.runtimeVersion, info.imageVariant)
 		if err != nil {
 			return "", err
 		}
 	}
 	return image, nil
 }
 
-// getSchedulerImageName returns the resolved scheduler image name for online `dapr init`.
-// It can either be resolved to the image-registry if given, otherwise GitHub container registry if
-// selected or fallback to Docker Hub.
-func getSchedulerImageName(imageInfo daprImageInfo, info initInfo) (string, error) {
```

Comment:
The return value is the `dapr` image, which contains all control plane services. So, we can just rename the method and avoid code duplication.